lib/find-view.js

```diff
@@ -0,0 +1,737 @@
+const _ = require('underscore-plus');
+const {TextBuffer, TextEditor, CompositeDisposable} = require('atom');
+const Util = require('./project/util');
+const etch = require('etch');
+
+module.exports =
+class FindView {
+  constructor(model, {findBuffer, replaceBuffer, findHistoryCycler, replaceHistoryCycler} = {}) {
+    this.model = model;
+    this.findBuffer = findBuffer;
+    this.replaceBuffer = replaceBuffer;
+    this.findHistoryCycler = findHistoryCycler;
+    this.replaceHistoryCycler = replaceHistoryCycler;
+    this.subscriptions = new CompositeDisposable();
+
+    etch.initialize(this)
+
+    this.findHistoryCycler.addEditorElement(this.findEditor.getElement());
+    this.replaceHistoryCycler.addEditorElement(this.replaceEditor.getElement());
+    this.handleEvents();
+    this.clearMessage();
+    this.updateOptionViews();
+    this.updateReplaceEnablement();
+    this.createWrapIcon();
+  }
+
+  update(props) {}
+
+  render() {
+    return (
+      etch.dom.div({tabIndex: -1, className: 'find-and-replace'},
+        etch.dom.header({className: 'header'},
+          etch.dom.span({ref: 'descriptionLabel', className: 'header-item description'},
```

Comment:
Oh, sorry. Should I have symlinked that branch to a local copy of the `etch` repo?